_maps/map_files/Delta/delta.dmm

```diff
@@ -96912,12 +96912,12 @@
 	name = "Dominator's";
 	req_access = list(1)
 	},
-/obj/item/gun/energy/dominator,
-/obj/item/gun/energy/dominator,
-/obj/item/gun/energy/dominator,
-/obj/item/gun/energy/dominator,
-/obj/item/gun/energy/dominator,
-/obj/item/gun/energy/dominator,
+/obj/item/gun/energy/dominator/sibyl,
```

Comment:
is it supposed to be here?

Reply:
kinda yeah. If I could, I would've just merge both PRs together. Anyway, that shouldn't make any problems

Reply:
> kinda yeah. If I could, I would've just merge both PRs together. Anyway, that shouldn't make any problems

kk